src/core_modules/capture-core/components/WidgetTwoEventWorkspace/OverflowMenu/Modal/UnlinkModal.tsx

```diff
@@ -63,20 +62,20 @@ export const UnlinkModal = ({
                 {i18n.t('Unlink event')}
             </ModalTitle>
             <ModalContent>
-                <p>{i18n.t('Are you sure you want to remove the link between these events? This action removes the link itself, but the linked event will remain.')}</p>
+                <p>{i18n.t('Are you sure you want to remove the link between these events? This action removes the link itself, but the linked event will remain.') as string}</p>
```

Comment:
Remove the `as string`. Remember to never add `as string` after the ` i18n.t(`) function